File: PrimeNumbers.c
#include <stdio.h>

int main(){

  int i = 2; // first prime number
  int num = 1; //number counter

  //while loop prints first 20 prime numbers
  while (num <= 20){
    int val = 0;
    // if i is divisible by any numbers val >1
    for (int j = 2;j < i; j++){
      if (i % j == 0){
	val++;
      }
      
    }
    //only numbers that are not divisible by any number are printed
    if (val == 0){
      printf("%i",i);
      printf("\n");
      num++;
    }
    i++;
  }
  return 0;
}

File: PrimeNumbers.py
import numpy as np

num = 2 #first prime number to check
prime = 1 #number of prime numbers calculated

while (prime <= 20):
    val = 0 #amount of factors
    #checks to see how many factors each number has
    for i in range(1,num):
        if(num % i == 0): val = val+1
        if (val >= 2): break #loop breaks if number has 2 or more factors
    #numbers with one factor are prime
    if (val == 1):
        print(num)
        prime = prime+1
    num = num+1
